### ISSUE TITLE
Feature Request: Transfer Products between Opportunities

### ISSUE THREAD

**Issue by b0ngw4ter** [CompositeLight/currentRMS-helper#27]
We often have repeat jobs with the exact same/similar kit which come back to the warehouse into a job-specific cage, and then goes back out a few days later.

Having experienced issues with current's transfer function - On these occasions if the technician confirms all the kit is ok, we manually transfer the items across to the new opportunity instead of checking everything in and then serialising and booking it back. 

Would there be a way to create something to allow both serialised and bulk kit and containerisation to be pulled to the new job from the old job?

**Comment by CompositeLight** [comment]
I agree the in built "transfer function" is confusing at best, and a danger to sanity at worst. Will need to look into what can be done on this. The API isn't really setup to do what you're asking - it would have to be an API call per item, rate limited to 100 per minute, which would take a while and could fail. I have an alternative idea though, will look into it when I get a chance.
